docs-css/docs-nav.css

```diff
@@ -1,96 +1,151 @@
-/* header */
-.navbar {
+.navbar-home {
+  padding: 0.7rem 0rem;
+  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 10%);
```

Comment:
You can use variables for these complex values like box-shadow so you can reuse it and have less chance of mistakes.